orchestrator-salt/src/main/resources/salt/pillar/postgresql/disaster_recovery.sls

```diff
@@ -1,2 +1,4 @@
 disaster_recovery:
   object_storage_url:
+  ranger_admin_group:
```

Comment:
@brycederriso Why was ranger_admin_group missing here?

Reply:
The values in this pillar aren't actually needed from what I can tell, so I forgot to add this when we did the first pass.

When the salt pillar update related code in CB runs, it _completely replaces_ the values in this file with the appropriate JSON values.

I think the intent of this file is to provide reasonable defaults so that we can make salt calls without first running salt pillar updates, but we're not doing that for backup/restore.

We can also kinda-sorta use these properties as documentation to show what we _will_ expect, and it makes it somewhat easier to experiment with salt scripts on a CB managed node -- you can manually edit the salt pillar and you have a starting key at least.